test/demo-logger-using-pvc/README.md

```diff
@@ -0,0 +1,52 @@
+# Containerized-Distributed-Logger 📝
+Logging Service to demonstrate the distributed logging with log collector.
+***
+
+### ⚽ Current Functionality:
+  - logger.py periodically logs the demo statements at /var/log/demoLogger
```

Comment:
> logger.py periodically logs the demo statements at /var/log/demoLogger

logger.py periodically dump sample logs in /var/log/demologger.log log file 

Also make note of extension to log file

Reply:
addressed. 👍